docs/applications/autodl.md

```diff
@@ -62,4 +62,13 @@ sudo systemctl disable irssi@<username>
 
 Configuration options for the GUI client can be found [here](https://code.google.com/archive/p/rutorrent/wikis/PluginAutodlirssi.wiki#Usage). The options will be found under the header "Usage".
 
-If you don't want to use the ruTorrent plugin and would prefer to configure the configuration files by hand, you can read the [autodl-irssi documentation](https://autodl-community.github.io/autodl-irssi/configuration/overview/) for an in-depth review of the available options.
\ No newline at end of file
+If you don't want to use the ruTorrent plugin and would prefer to configure the configuration files by hand, you can read the [autodl-irssi documentation](https://autodl-community.github.io/autodl-irssi/configuration/overview/) for an in-depth review of the available options.
+
+## Patch for 'Could not read: SSL: Unknown error code: 5'
```

Comment:
Could this be rewritten into the `## Troubleshooting` format with the snippet import? Might as well not lose this